dbos/__init__.py

```diff
@@ -1,4 +1,5 @@
 from . import error as error
+from .context import DBOSContextEnsure as DBOSContextEnsure
```

Comment:
Why do we need 'DBOSContextEnsure as DBOSContextEnsure` here (and similar clauses in the following imports)

Reply:
You mean get rid of the 'as ...' part?

Reply:
I think we do this because if we change the internal names they won't cause breaking changes to users.

Reply:
We would add those in cases where we change internal names to avoid breaking changes to users.  As they sit, I agree with Harry that they don't serve a purpose.  I also found vs.code renames both parts of the clause automatically if you hit "rename symbol", so that's no help at all!